Helper/Order.php

```diff
@@ -1403,4 +1425,23 @@ protected function isAnAllowedUpdateFromAdminPanel(OrderInterface $order, $trans
                in_array($transactionState, [self::TS_AUTHORIZED, self::TS_COMPLETED]) &&
                $order->getState() === OrderModel::STATE_PAYMENT_REVIEW;
     }
+
+    /**
+     * @param OrderPaymentInterface $payment
+     *
+     * @return \Magento\Framework\Phrase|string
+     */
+    protected function getVoidMessage(OrderPaymentInterface $payment)
+    {
+        $authorizationTransaction = $payment->getAuthorizationTransaction();
```

Comment:
We already got `$authorizationTransaction` in the switch statement - can we pass it into this function instead of getting it again?

Reply:
@johnmccombs1 Is the cost of `$payment->getAuthorizationTransaction();` another database lookup?